developer-docs-site/static/messageboard_demo/client/README.md

```diff
@@ -0,0 +1,187 @@
+# MessageBoard
+
+### Setup and Run the Demo
+
+Download Aptos Core
+
+```bash
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core
+./scripts/dev_setup.sh
+source ~/.cargo/env
+```
+
+Install Apto Cli [[doc](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos)]
+
+```bash
+cargo install --git https://github.com/aptos-labs/aptos-core.git aptos
+```
+
+Run demo using the commands below 
+
+```bash
+cd developer-docs-site/static/messageboard_demo/client
+yarn install
+yarn demo
+```
+
+**This demo shows**
+
+1. write a basic smart contract with move [[source](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/messageboard)]
+2. publish, create and interact with messageboard [[source](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/static/messageboard_demo/client)]
```

Comment:
- publish → Publish
- End the sentence with a period. (see item 1)

Reply:
updated